CMakeLists.txt

```diff
@@ -57,56 +55,28 @@ if(USE_ESSLBM)
 endif()
 
 add_library(lbmLib)
+target_compile_features(lbmLib PRIVATE cxx_std_17)
 
-set(EXTERNAL_DIR ${lbm_SOURCE_DIR}/external)
-
-set(EXTERNAL_HEADER_LIST
-    ${EXTERNAL_DIR}/tinyxml/tinyxml.h
-    ${EXTERNAL_DIR}/tinyxml/tinystr.h
-    ${EXTERNAL_DIR}/zlib/crc32.h
-    ${EXTERNAL_DIR}/zlib/deflate.h
-    ${EXTERNAL_DIR}/zlib/gzguts.h
-    ${EXTERNAL_DIR}/zlib/inffast.h
-    ${EXTERNAL_DIR}/zlib/inffixed.h
-    ${EXTERNAL_DIR}/zlib/inflate.h
-    ${EXTERNAL_DIR}/zlib/inftrees.h
-    ${EXTERNAL_DIR}/zlib/trees.h
-    ${EXTERNAL_DIR}/zlib/zconf.h
-    ${EXTERNAL_DIR}/zlib/zlib.h
-    ${EXTERNAL_DIR}/zlib/zutil.h
+set(LBM_HEADER_LIST
+    incl/unistd.h
 )
 
-set(EXTERNAL_SOURCE_LIST
-    ${EXTERNAL_DIR}/tinyxml/tinyxml.cpp
-    ${EXTERNAL_DIR}/tinyxml/tinystr.cpp
-    ${EXTERNAL_DIR}/tinyxml/tinyxmlerror.cpp
-    ${EXTERNAL_DIR}/tinyxml/tinyxmlparser.cpp
```

Comment:
The `tinyxml` library sources unfortunately seem to be necessary for the definitions of the class `TiXmlDocument` which is used in OpenLB.
This looks like such a pain because OpenLB can only be built via Makefiles. :(